values.yaml

```diff
@@ -89,3 +89,20 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+postgresql:
+  enabled: true
+  primary:
+    initdb:
+      scripts:
+        create-db.sql:
+          CREATE DATABASE grader_config;
```

Comment:
I would call the db something like `eduhelx` instead, since it's containing state about the entire eduhelx system, not just configuration and not just about ottergrader.

Reply:
ok, though eduhelx seems overly broad. maybe eduhelx_config?

Reply:
the name of the db describes what the db is holding the state of. it's not just holding the state of the configuration, it's also holding student information and submission information, which is why I think we should just go with `eduhelx`